config/site/site.config.js

```diff
@@ -38,13 +38,6 @@ const siteConfig = {
    */
   hotReloading: true,
 
-  /* The mono-repo package path(s). This is used to automatically alias package directories in webpack. We can support multiple. */
```

Comment:
This is technically non passive but we don't use the setting, nor does anyone in our internal repo.

Reply:
Should we still add it to the changelog?

Reply:
no harm in it. While i'm here i'm going to remove another bit of config no one is using. "webpackAliasOptions"